src/legacy/ui/public/vis/editors/config/editor_config_providers.ts

```diff
@@ -78,7 +78,7 @@ class EditorConfigProviderRegistry {
     current: EditorParamConfig,
     merged: EditorParamConfig,
     paramName: string
-  ): { fixedValue?: any; base?: number } {
+  ): { fixedValue: unknown } | { base: number } | {} {
```

Comment:
We know this method is returning either of those, so we use a union type here instead, which makes more sense.